core/src/main/scala/org/apache/spark/scheduler/cluster/cook/CoarseCookSchedulerBackend.scala

```diff
@@ -115,6 +118,11 @@ private[spark] class CoarseCookSchedulerBackend(
   }
 
   var runningJobUUIDs = Set[UUID]()
+  var executorsToJobs = HashMap[String, UUID]()
```

Comment:
Can you rename this to `executorsToJobIds`? I had to scroll up here to ensure that it was string -> UUID.